### ISSUE TITLE
Duplicate ToC output when using dynamic ToCs

### ISSUE THREAD

**Issue by schwamborn** [eclipse/sumo#6686]
Example:

```
<DYNTOR lanePos="0.13" lane="approach2_1" t="651.60" id="CAVGB.38"/>
<TOR lanePos="0.13" lane="approach2_1" t="651.60" id="CAVGB.38"/>
```

**Comment by namdre** [comment]
The duplicate output is back. The fixes to #6755 somehow caused this but I could not figure out why. 
Please check.

**Comment by namdre** [comment]
Not sure what happened. In today's tests the duplicate disappeared again and I cannot reproduce it locally either.
